upgrade_guide/migrating_from_astyanax/language_level_changes/README.md

```diff
@@ -0,0 +1,66 @@
+# Language change : from Thrift to CQL
+The data model changes when using *CQL* (Cassandra Query Language).
+*CQL* is providing an abstraction of the low-level data stored in *Cassandra*, in
+opposition to *Thrift* that aims to expose the low-level data structure directly.
+[But note that this changes with Cassandra 3’s new storage engine.](http://www.datastax.com/2015/12/storage-engine-30)
+
+*Thrift* exposes *Keyspaces*, and these *Keyspaces* contain *Column Families*. A
+*ColumnFamily* contains *Rows* in which each *Row* has a list of an arbitrary number
+of column-values. With *CQL*, the data is **tabular**, *ColumnFamily* gets viewed
+as a *Table*, the **Table Rows** get a **fixed and finite number of named columns**.
```

Comment:
In one case Table Rows is bolded for emphasis, and in another case it's underlined, is that intentional?